src/components/Chat/ChatIndicator.tsx

```diff
@@ -52,16 +52,12 @@ function onChatSubscribeGroupMentionsChange(pref) {
     chat_subscribe_new_group_chat_mentioned = pref;
     event_emiter.emit("subscription_changed");
 }
-let chat_subscribe_new_tournament_chat_messages = false;
 preferences.watch("chat-subscribe-tournament-chat-unread", onChatSubscribeTournamentMessageChange);
-function onChatSubscribeTournamentMessageChange(pref) {
-    chat_subscribe_new_tournament_chat_messages = pref;
```

Comment:
This one actually does look like a bug to me.  See:

https://github.com/online-go/online-go.com/blob/29b7a1160d6634fa66c9fa0f3b93d201f7ee8b79/src/components/Chat/ChatIndicator.tsx#L68-L79

`chat_subscribe_new_tournament_chat_messages` is returned twice!

Reply:
Tagging @flovo for more context.

Reply:
I'm 99% sure that's a bug, so yeah let's not remove that variable and instead fix the usage.

Reply:
Another good catch, went ahead and updated